game/src/main/scala/io/aigar/game/Entity.scala

```diff
@@ -5,6 +5,7 @@ import com.github.jpbetz.subspace.Vector2
 trait Entity {
   var position: Vector2
   protected var _mass = 0f
+  val score: Int
```

Comment:
Could we add a comment to indicate that this is *added* to the score rather than a score counter? Or rename if you find a better one?

I love your work.